app/scripts/start.js

```diff
@@ -6,7 +6,7 @@
  */
 const { spawn } = require('child_process');
 
-const serverProcess = spawn('cd ../ && make backend && make run-backend', [], {
+const serverProcess = spawn('cd ../ && make backend && make run-backend APP_MODE=true', [], {
```

Comment:
Did you confirm this works? I think you need to add "APP_MODE=true'" before make run-backend?

Reply:
@illume yes I have tried it myself. Either way works. Passing before make run-backend also works and passing after it also works. Should I pass before it?